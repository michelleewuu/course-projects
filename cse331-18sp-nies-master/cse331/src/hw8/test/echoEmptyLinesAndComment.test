# Check the program echo empty lines and 
# lines starting with # character.

# Is this working correctly?

# terminate program
q