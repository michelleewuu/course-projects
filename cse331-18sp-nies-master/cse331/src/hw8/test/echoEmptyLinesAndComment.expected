Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Check the program echo empty lines and 
# lines starting with # character.

# Is this working correctly?

# terminate program